core/src/main/java/com/exadel/aem/backpack/request/impl/RequestAdapterImpl.java

```diff
@@ -369,7 +511,11 @@ private boolean isSupportedType(Class<?> clazz) {
      */
     private static <T> T createDefaultObject(final Class<T> tClazz) {
         try {
-            return tClazz.getConstructor().newInstance();
+            T newInstance = tClazz.getConstructor().newInstance();
+            if (newInstance == null) {
```

Comment:
This condition is not needed, because if there's no default constructor, the exception will have been thrown in the previous line already